src/main/java/tk/sciwhiz12/concord/Concord.java

```diff
@@ -63,15 +75,27 @@ public class Concord {
     public Concord() {
         ModLoadingContext.get().registerExtensionPoint(IExtensionPoint.DisplayTest.class,
                 () -> new IExtensionPoint.DisplayTest(() -> NetworkConstants.IGNORESERVERONLY, (ver, remote) -> true));
-        ModPresenceTracker.register();
 
         ConcordConfig.register();
 
+        DistExecutor.unsafeRunWhenOn(Dist.CLIENT, () -> ConcordClient::setup);
```

Comment:
This should be treated as an implementation detail and changed to a FMLLoader.getDist check.